### DISCUSSION TITLE
qbittorrent-libtorrent-v1 4.6.0-1 maintainer?

### DISCUSSION THREAD

**Discussion by 3k2** [ouuan/ouuan#44] [Q&A (English)] [answered]
Hello!

Do you think you could be interested to maintain qbittorrent-libtorrent-v1 4.6.0-1 AUR package again? at least in my case I still have the issue where qbittorrent isn't working properly with the latest libtorrent v2 it performs much worse compared to v1. I'm sure there are many others who still have the same issue. 

[AUR](https://aur.archlinux.org/packages/qbittorrent-libtorrent-v1)

**Comment by ouuan** [comment, marked as answer]
OK, I adopted this package again.
